src/UnitTests/BeforeAfterMapping.cs

```diff
@@ -33,6 +33,24 @@ public void Before_and_After_should_be_called()
             afterMapCalled.ShouldBeTrue();
         }
 
+        [Fact]
+        public void Before_and_After_overrides_should_be_called()
```

Comment:
These tests seem to call the generic versions, so they are not relevant. Right? :)

Reply:
Note to myself: stop making PR at 1AM. I'll fix that tonight if I have time

Reply:
Fixed it